src/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```

Comment:
Remove service worker from this file and also remove all useless code.

Reply:
Serviceworker file and all useless code has been removed.